dependencies.txt

```diff
@@ -29,6 +29,9 @@ conda-forge::descartes=1.1.0     # from geopandas; have not verified we need to
 conda-forge::nodejs=9.11.1       # for bokeh (not a dep of bokeh conda package)
 conda-forge::boost-cpp=1.66.0    # current c-f pin (used by various deps including filigree)
 
+# libgdal 2.2.4 build 5 (pinned for compatibility)
+pyviz/label/earthsim::libgdal
+
```

Comment:
Generally I'd favor leaving the channel the same and pinning to the previous version (2.2.3), rather than working with a version (2.2.4) that has multiple flavors that appear to be the same but some work and some don't.  I assume there's a good reason in this case.

Reply:
Just that I would have to go back and evaluate how 2.2.3 was built, and whether or not it actually works (with the rest of our packages) - plus there could be multiple builds of that version too, anyway. (Whereas going back to the build we've previously been using, which is known to work, is much less time consuming.)

So I agree with you, but won't change anything unless you request it. Probably wouldn't turn out to be a big deal, but it's just a bit of an unknown quantity (to me) in advance of actually doing it.

Reply:
There may well be multiple builds of 2.2.3, but we are unlikely to get anything but the latest build, and other newer builds are less likely to appear for a previous version than for a new version, so rolling back to 2.2.3 is what I'd prefer to happen in general.  Sure, it might not work, in which case an approach like this will always work, but it seems very likely to work in most cases, and is less complex, so I'd vote for trying that approach first next time, and choosing the nuclear option only if forced.  This time, it's done, so don't worry about it!

Reply:
(2.2.4 itself is a previous version, with a more recent build than the current version)

Reply:
Sigh.